cypress/integration/gift_cards.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {HomePo} from "../support/home.po";
+import {GiftCardsPo} from "../support/giftCards.po";
+
+describe('Gift cards suite', () => {
+    const homePage = new HomePo();
+    const giftCardPage = new GiftCardsPo();
+
+    beforeEach(() => {
+
+        cy.visit('');
+    })
+
+    it('should ', () => {
+        const price = '100';
+        const giftCardId = '[href*="/B07TMMZKBJ"]';
```

Comment:
And because you're using this ID for the assertion, that this product was added to the bucket, then you can just get this product, save to alias and then check that this element with an alias was added.